consensus/block.go

```diff
@@ -1,24 +1,23 @@
-package consensus
+package consensus // TODO (Olshansk) needs a README file with proper code structure
```

Comment:
NIT: `// TODO (Olshansk) needs` -> ` // TODO(olshansky): needs`

Can you put this in a comment in the main README?